Classes/KIFUIViewTestActor.m

```diff
@@ -263,9 +283,10 @@ - (void)enterTextIntoCurrentFirstResponder:(NSString *)text fallbackView:(UIView
 - (void)setText:(NSString *)text;
 {
     KIFUIObject *found = [self _predicateSearchWithRequiresMatch:YES mustBeTappable:NO];
```

Comment:
Yea, ObjC doesn't really have any sort of nullability enforcement, it just relies on nil messaging semantics and assumes that you're ok with everything being nullable.